src/main/java/DeadlineTask.java

```diff
@@ -0,0 +1,30 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class DeadlineTask extends Task{
+    private LocalDate deadline;
+
+    /**
+     * Constructor for deadline task object
+     * @param name description of the task
+     * @param isCompleted is it completed or not
+     * @param deadline deadline for the task
+     */
+    public DeadlineTask(String name, boolean isCompleted, String deadline) {
+        super(name,isCompleted);
```

Comment:
Maybe you could add a space after "name," so that it follow the java coding standards?
E.g: super(name, isCompleted)